sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -86,6 +88,98 @@ public TimeSeriesInsightsClient(string environmentFqdn, TokenCredential credenti
             _timeSeriesTypesRestClient = new TimeSeriesTypesRestClient(_clientDiagnostics, _httpPipeline, environmentFqdn, versionString);
         }
 
+        /// <summary>
+        /// Gets model settings asynchronously.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>The Model Settings which includes model display name, Time Series ID properties and default type ID with the http response <see cref="Response{T}"/>.</returns>
```

Comment:
unless referring to a type by name and with a seealso reference, you don't need to capitalize them, so this should be `model settings`.